README.md

```diff
@@ -12,6 +12,10 @@
 - CSS
 
 
+desktop-version
+
+
+ master
 
```

Comment:
Great work @fullstop125 🤝

- [ ] [Optional] You may please remove this master statement. Try to customize your readme file accordingly.

Reply:
resolved